### PR TITLE
Add README and bump package to 0.0.6

### PR DESCRIPTION
**Pull request by bvaughn**
- [x] Remove `upload-sourcemaps` alias (subjective change)
- [x] Add basic `README` and `Contributing` files
- [x] Bump package version to 0.0.6

### CODE REVIEW

File: packages/replayio/src/commands/upload-source-maps.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ import { dim } from "../utils/theme";
 import { logPromise } from "../utils/async/logPromise";
 
 registerCommand("upload-source-maps <paths...>", { requireAuthentication: true })
-  .alias("upload-sourcemaps")
```
Comment (bvaughn) [inline packages/replayio/src/commands/upload-source-maps.ts:10]: Subjective change, I guess, but this...
Screenshot 2024-04-12 at 10 05 55 AM

...is a better first impression than this...
Screenshot 2024-04-12 at 10 04 16 AM

This isn't a command people will be running often, so I think we can just ask them to type it correctly :)

Comment (Andarist) [reply packages/replayio/src/commands/upload-source-maps.ts:10]: The only alternative that comes to mind is to drop the aliases from this table altogether (see my comment [here](https://github.com/replayio/replay-cli/pull/375#discussion_r1562613521)) - that is if we want to keep it being available to people.

**Review by Andarist — APPROVED** [review]

File: packages/replayio/src/commands/upload-source-maps.ts (hunk 1 above)
Comment (bvaughn) [reply packages/replayio/src/commands/upload-source-maps.ts:10]: I think we should just do a clean break. We've already done this with so many things (`ls` -> `list`, `rm` -> `remove` and `rm-all` -> `remove --all`, etc.) I don't see a reason to carry the old alias forward :)

Comment (Andarist) [reply packages/replayio/src/commands/upload-source-maps.ts:10]: My thinking here was a mix of keeping the old name and that people (us included :P) use that spelling interchangeably - it's actually the latter that made me add the alias. I'm fine with removing it though 👍

**Pull request merged**